src/pygrambank/sheet.py

```diff
@@ -60,20 +60,20 @@ def _require_comment(feat, reason):
         == '0'
     ):
         reason = 'GB333, GB334, GB335, and GB336 are all 0'
-        for feat in ('GB333', 'GB334', 'GB335', 'GB336'):  # pragma: nocover
```

Comment:
what did "pragma: no cover" mean?

Reply:
The test suite creates a coverage report at the end that says which lines of code were run during testing and which were not.  This comment just tells the test suite ‘no need to report that line -- we have our reason for why we don't test it’ (although in this case I don't remember what my reason actually was back then (<\_<)" ).

Now the lines *are* being tested properly so I removed the comment.

Reply:
Ah, makes sense :)